Added client-side commands and server type parameter for commands (#3995)
MCreator/MCreator: plugins/mcreator-localization/help/default/command/type.md

This option allow you to determine the type of the command and how/where it can be used.

> * **Standard**: The command can be used everywhere. This is the normal and ancient behaviour (e.g. `/give`).

Reviewer: I would avoid the use of ancient as this implies something being old or almost deprecated which this is not.

You should call this default/standard behaviour

Reviewer 2: **Common** would fit too I think 🤔
```suggestion
* **Standard**: The command can be used everywhere. This is the common behavior for most vanilla commands (e.g. `/give`).
```
(If I'm not lying about _most vanilla commands_)

> * **Standard**: The command can be used everywhere. This is the common/normal behaviour (e.g. `/give`).
> * **Multiplayer only**: The command will be available on multiplayer servers only (e.g. `/ban`)
> * **Singleplayer only**: The command will be available on singleplayer worlds only (e.g. `/publish`)

Reviewer 2: ```
Эта опция указывает тип данной команды и допустимую область её использования.
* **Standard**: Команда доступна без ограничений. Стандартное поведение команд (например, `/give`).
* **Multiplayer only**: Команда доступна только в многопользовательском режиме (например, `/ban`).
* **Singleplayer only**: Команда доступна только в одиночном режиме (например, `/publish`).
* **Client-side**: Команда доступна только на стороне клиента, вызываемые ей действия не могут затронуть сторону сервера.
```
```suggestion
* **Standard**: The command can be used everywhere. This is the common/normal behavior (e.g. `/give`).
* **Multiplayer only**: The command will be available on multiplayer servers only (e.g. `/ban`).
* **Singleplayer only**: The command will be available on singleplayer worlds only (e.g. `/publish`).
```
